lib/src/main/AndroidManifest.xml

```diff
@@ -37,21 +37,21 @@
             android:launchMode="singleTask"
             android:screenOrientation="portrait"
             android:theme="@style/Lock.Theme">
-            <intent-filter>
-                <action android:name="android.intent.action.VIEW" />
+            <intent-filter android:autoVerify="true">
+            <action android:name="android.intent.action.VIEW" />
 
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
 
                 <data
                     android:host="${auth0Domain}"
                     android:pathPrefix="/android/${applicationId}/email"
-                    android:scheme="${auth0Scheme}" />
+                    android:scheme="https" />
```

Comment:
the scheme is not customizable anywhere (API nor dashboard), so there's no reason to dynamically set one.